### PR TITLE
[WKP-2116] Add docs on how to install standalone MCCP

### PR DESCRIPTION
**Pull request by sarataha**
A guide on how to install a standalone MCCP via MCCP helm chart

### CODE REVIEW

**Comment by foot** [comment]
This is looking great! ⭐ . 

https://github.com/weaveworks/wkp-docs/blob/WKP-2116/standalone-mccp-docs/docs/mccp/database-configuration.md could also use some small tweaks

File: docs/mccp/usage-guide.md [hunk 1]
```diff
@@ -24,92 +26,84 @@ node/ip-192-168-68-212.eu-north-1.compute.internal
...
-Now you can enable the MCCP in config.yaml.
+This label will ensure that the MCCP pods will be deployed and run on the selected node, if the **SQLite** option has been chosen.
 
 :::info
-_The MCCP also supports **PostgreSQL** or mounting SQLite on Persistent Volume instead of the host volume described here. See [Database Configuration](./database-configuration) for details._
+_MCCP also supports **PostgreSQL** or mounting SQLite on Persistent Volume instead of the host volume described here. See [Database Configuration](./database-configuration) for details._
 :::
 
-### 2. Enable the MCCP in config.yaml
+### 2. Creating a secret for docker repository
```
Comment (foot) [inline docs/mccp/usage-guide.md:35]: Should we add _Create a namespace_

e.g. 

Run `kubectl create namespace mccp` somewhere too? 🤔

File: docs/mccp/usage-guide.md [hunk 2]
```diff
@@ -24,92 +26,84 @@ node/ip-192-168-68-212.eu-north-1.compute.internal
...
 
-To enable the MCCP feature edit the `setup/config.yaml` file, set the `enabledFeatures: fleetManagement`
-field to `true`, commit and push to the cluster repository.
+Create a secret that contains your docker repository credentials. This secret will be used by Kubernetes during deployment in order to pull down the MCCP images. You can find instructions on how to generate this secret [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
```
Comment (foot) [inline docs/mccp/usage-guide.md:37]: Maybe add a, "for example":
```

            kubectl create secret docker-registry docker-io-pull-secret \
              --namespace mccp \
              --docker-username="${DOCKER_IO_USER}" \
              --docker-password="${DOCKER_IO_PASSWORD}" 
```

File: docs/mccp/usage-guide.md (hunk 1 above)
Comment (yiannistri) [reply docs/mccp/usage-guide.md]: Added step for creating a namespace 👍

File: docs/mccp/usage-guide.md (hunk 2 above)
Comment (yiannistri) [reply docs/mccp/usage-guide.md:106]: Added 👍

**Review by dinosk — APPROVED** [review]
💯 I really like the tabs for the different installation options.

**Pull request merged**